tests/system/python/conftest.py

```diff
@@ -668,6 +669,33 @@ def _disable_sch(fledge_url, sch_name):
     return _disable_sch
 
 
+@pytest.fixture(scope="function", autouse=True)
+def collect_support_bundle(request):
+    def _collect_support_bundle():
+        PROJECT_ROOT = subprocess.getoutput("git rev-parse --show-toplevel")
+        try:
+            conn = http.client.HTTPConnection("{}".format(url))
+            conn.request("POST", "/fledge/support")
+            r = conn.getresponse()
+            assert 200 == r.status
+            r = r.read().decode()
+            jdoc = json.loads(r)
+            assert jdoc["bundle created"]
+            subprocess.run(["mkdir -p {0}/support/ && cp -r {1} {0}/support/.".format(PROJECT_ROOT, jdoc["bundle created"])], shell=True, check=True)
```

Comment:
In case of package based system tests, what will be the PROJECT_ROOT? Are support bundles stored in PROJECT_ROOT path?
What is the value for jdoc["bundle created"] ?